lib/internal/fs/utils.js

```diff
@@ -649,6 +650,16 @@ const getValidatedPath = hideStackFrames((fileURLOrPath, propName = 'path') => {
   return path;
 });
 
+const getValidatedFd = hideStackFrames((fd, propName = 'fd') => {
+  if (ObjectIs(fd, -0)) {
+    fd = 0;
+  }
+
+  validateInt32(fd, propName, 0);
+
+  return fd;
```

Comment:
I would do this instead.

```suggestion
  validateInt32(fd, propName, 0);
  return (fd | 0);
```

Reply:
Can you lay out the reasons why you think this is better? With the suggestion from mscdex above, we can avoid the `validateInt32` call, you suggestion seems less efficient at a first glance.

Reply:
Well, I was actually considering just using  `(fd | 0);` expression cuz' I consider it's a bit simpler and gets its job done (avoid the crash, the actual concern of the issue).

BTW, This is not a blocker, just a suggestion.

Reply:
> BTW, This is not a blocker, just a suggestion.

FYI "Request Changes" GitHub feature is used to block the PR from landing: 

https://github.com/nodejs/node/blob/9cf23414d5883b1aabcfe5bddeb8a23396064f2e/doc/guides/collaborator-guide.md#L154-L161

You can use the "Comment" (or even "Approve") feature for non-blocking suggestions.